server/lib/tuist_web/marketing/components/marketing_layout_components/navbar.html.heex

```diff
@@ -117,10 +123,16 @@ resources_items = [
     href: "https://github.com/tuist"
   },
   %{
-    icon: :news,
-    title: dgettext("marketing", "Newsletter"),
-    subtitle: dgettext("marketing", "Stay updated—subscribe to our newsletter"),
-    href: TuistWeb.Marketing.MarketingHTML.localized_href("/newsletter")
+    icon: :dashboard,
+    title: dgettext("marketing", "Public dashboard"),
+    subtitle: dgettext("marketing", "Explore Tuist's features on the public dashboard"),
+    href: "/tuist/tuist"
+  },
+  %{
+    icon: :users,
+    title: dgettext("marketing", "Case studies"),
+    subtitle: dgettext("marketing", "See how leading teams scale with Tuist"),
+    href: TuistWeb.Marketing.MarketingHTML.localized_href("/case-studies")
   }
 ] %>
```

Comment:
This wasn't designed, and since @asmitbm is away, I'd suggest that we go ahead with this one and then adjust it once we have the final design.